pkg/operator/collection.go

```diff
@@ -263,6 +263,9 @@ func (r *collectionReconciler) makeCollectorDaemonSet(spec *monitoringv1alpha1.C
 	for _, matcher := range spec.Filter.MatchOneOf {
 		collectorArgs = append(collectorArgs, fmt.Sprintf("--export.match=%s", matcher))
 	}
+	if r.opts.OperatorEnv != "" {
+		collectorArgs = append(collectorArgs, fmt.Sprintf("--export.user-agent=Prometheus-Collector/%s/%s", Version, r.opts.OperatorEnv))
```

Comment:
Can we make sure that the UA is a variation of the regular one (https://github.com/GoogleCloudPlatform/prometheus-engine/blob/main/cmd/rule-evaluator/main.go#L84).

Like just reproduce what is there and then add environment in braces behind that?